singer/src/test/java/com/pinterest/singer/kubernetes/TestKubeService.java

```diff
@@ -99,32 +99,55 @@ public void testInit() throws IOException {
     }
 
     @Test
-    public void testGoodPodFetch() throws KeyManagementException, ClientProtocolException, NoSuchAlgorithmException,
-            KeyStoreException, MalformedURLException, IOException {
+    public void testGoodPodFetch()
+        throws KeyManagementException, ClientProtocolException, NoSuchAlgorithmException,
+               KeyStoreException, MalformedURLException, IOException {
         registerGoodResponse();
 
         KubeConfig kubeConfig = new KubeConfig();
         KubeService poll = new KubeService(kubeConfig);
         Set<String> fetchPodNamesFromMetadata = poll.fetchPodNamesFromMetadata();
 
         // check uid count is correct
-        assertEquals(12, fetchPodNamesFromMetadata.size());
-        
-        Map<String, String> podToNamespaceMap = new HashMap<>();
-        podToNamespaceMap.put("enimanager-ppl56", "default");
-        podToNamespaceMap.put("zk-update-monitor-r1vlt", "default");
-        podToNamespaceMap.put("tcollector-q4qx8", "default");
-        podToNamespaceMap.put("metrics-agent-8vm9g", "default");
-        podToNamespaceMap.put("kubernetes-dashboard-1835568627-hhfhj", "kube-system");
-        podToNamespaceMap.put("test-ci-0", "kubernetes-plugin");
-        for (Entry<String, String> entry : podToNamespaceMap.entrySet()) {
-          String name = entry.getKey();
-          String namespace = entry.getValue();
-          assertTrue(fetchPodNamesFromMetadata.contains(name));
-          assertTrue(fetchPodNamesFromMetadata.contains(namespace + "_" + name));
+        assertEquals(6, fetchPodNamesFromMetadata.size());
```

Comment:
@jfzunigac should the test json be updated as well and a unit test trigged with it?

Reply:
The test json works for this use case as well, I included minor changes to one of the test cases that relies on the json